model/sparql/GenericSparql.php

```diff
@@ -542,34 +542,37 @@ public function queryConceptInfo($uris, $arrayClass = null, $vocabs = array(), $
      * @param string $lang
      * @return string sparql query
      */
-    private function generateQueryTypesQuery($lang) {
-        $fcl = $this->generateFromClause();
+    private function generateQueryTypesQuery(string $lang): string {
+        $vocabs = $this->model->getVocabularies();
```

Comment:
If there are no vocabs, then the query will return all the types from all the graphs. Would it be better to return an empty result int that case?